js-module-base/pom.xml

```diff
@@ -2,15 +2,17 @@
 
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
     <modelVersion>4.0.0</modelVersion>
+    
     <parent>
-        <groupId>org.jenkins-ci.plugins</groupId>
-        <artifactId>plugin</artifactId>        
-        <version>2.30</version>
-        <relativePath/>
+        <groupId>org.jenkins-ci.ui</groupId>
+        <artifactId>jenkins-js-libs</artifactId>
+        <version>2.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
```

Comment:
So, could this POM material just be moved up into the root POM to simplify things?